### ISSUE TITLE
[COMPATIBILITY]: Fan Control on Legion 5 15ACH6H

### ISSUE THREAD

**Issue by einfachGaer** [BartoszCichecki/LenovoLegionToolkit#708]
### Version

2.12.0

### OS

Windows 11 Pro 22h2 22621.1635

### Device

Legion 5 15ACH6H

### BIOS version

GKCN60WW

### Which features of the application did you test?

I was looking for fan controlls within the custom power mode settings, sadly I can only force maximum fan speed without defining a curve.

![image](https://user-images.githubusercontent.com/116717428/236405776-1ef73f2e-f48f-4b55-8097-f601ad5646fb.png)

### Which features of the application do not work as expected?

Shouldn't be the Legion 5 15ACH6H model supported?

If there are hardware limitations feel free to close this.

**Comment by BartoszCichecki** [comment]
Yup it's a hardware limitation. Fan curve is available on gen 7 and 8.
